src/etl/adapter/repository.py

```
@@ -43,26 +56,35 @@ async def save_batch(self, messages: List[MessageMetadata]) -> None:
                     id=str(uuid.uuid4()),
                     vector=vector.tolist(),
-                    payload=asdict(msg)
+                    payload={
+                        "chat_id": str(msg.chat_id),
+                        "sender_id": str(msg.sender_id),
+                        # Если текста нет, сохраняем None или пустую строку, не приводя тип вслепую
+                        "text": msg.text if msg.text is not None else "", 
+                        "attached_files": msg.attached_files
+}
                 )
                 for vector, msg in zip(embeddings, messages)
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)